acprep

```diff
@@ -535,7 +535,24 @@ class PrepareBuild(CommandLineApp):
                         'sudo', 'apt-get', 'install',
                         'build-essential',
                     ]
-                    if release == 'jammy':
+                    if release == 'noble':
+                        packages.extend([
+                            'doxygen',
+                            'cmake',
+                            'ninja-build',
+                            'zlib1g-dev',
+                            'libbz2-dev',
+                            'python3-dev',
+                            'libgmp3-dev',
+                            'libmpfr-dev',
+                            'gettext',
+                            'libedit-dev',
+                            'texinfo',
+                            'lcov',
+                            'libutfcpp-dev',
+                            'sloccount'
+                        ])
+                    elif release == 'jammy':
```

Comment:
There seems to be a bit of code duplication here, @xudon9, would you be willing to come up with improved handling of the shared packages between distributions and distribution versions?

Reply:
Done

Reply:
Much appreciated, @xudon9, this looks very promising 👍 Let's see if CI runs through…